api/graphql/schema.graphql

```diff
@@ -873,6 +904,79 @@ input UpdateSkillInput {
     keywords: [String!]
 }
 
+input WorkExperienceMutationInput {
```

Comment:
The word Mutation is redundant in these names, as graphql `inputs` are almost all used in mutations.